public/utils/utils.tsx

```diff
@@ -163,3 +165,17 @@ export const getTitleWithCount = (title: string, count: number | string) => {
     </EuiTitle>
   );
 };
+
+export function convertTimestampToString(timestamp: number | string) {
+  if (typeof timestamp === 'string') {
+    return timestamp;
+  }
+  return moment(timestamp).format();
+}
+
+export function convertTimestampToNumber(timestamp: number | string) {
```

Comment:
This method is not being used currently ?

Reply:
It is actually being used in the create historical detector page when converting Formik values to the numbers when making a create/update detector request. It's not included in this PR, but you can see it used [here](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/349/files)